main.rb

```diff
@@ -1,12 +1,25 @@
+# main.rb
+
 require_relative 'app'
 require_relative 'book'
 require_relative 'student'
 require_relative 'rental'
 require_relative 'teacher'
 require_relative 'menu'
+require_relative 'storage'
+require_relative 'person'
 
 def main
+  # Load people data
+  people = Storage.read_data(Storage::PEOPLE_FILE_PATH, Person)
```

Comment:
👍 Good progress so far! 👍 However, there's a small adjustment to enhance the structure of your code:

- [ ] **Implementing main.rb File:** In the [previous milestone](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_refactor.md#:~:text=Pay%20special%20attention,or%20to%20quit), you learned about the concept of having a `main.rb` file. The role of this file is to start your code, ask the user for the initial choice of options, and handle the flow of the program. Please consider updating your code to follow this instruction by implementing **Single-responsibility Principle**. It will improve the organization and clarity of your project.

By implementing the **Single-responsibility Principle**, you'll adhere to the recommended structure for better code organization. Keep up the good work! 🚀📄